grants.yml

```diff
@@ -312,6 +312,8 @@
     - queue:create-task:low:lambda/gecko-t-*
     - queue:create-task:low:scriptworker-prov-v1/depsigning-mac-v1
     - queue:create-task:low:scriptworker-prov-v1/depsigning-mac-v1-dev
+    - queue:create-task:low:scriptworker-prov-v1/dep-gecko-signing-mac14m2
+    - queue:create-task:low:scriptworker-prov-v1/gecko-signing-mac14m2
```

Comment:
This looks like it's giving prod signing access to l1 repos?

Reply:
Nice catch! Thanks!